src/frametree.h

```diff
@@ -34,7 +34,7 @@ class FrameTree : public std::enable_shared_from_this<FrameTree> {
     bool focusClient(Client* client);
     //! focus a frame within its tree
     static void focusFrame(std::shared_ptr<HSFrame> frame);
-    bool focusInDirection(Direction dir, bool externalOnly);
```

Comment:
I think we agreed on camelCase as the canonical style (even though it's not consistently used yet)

Reply:
Fixed